airbyte-cdk/java/airbyte-cdk/core/src/testFixtures/kotlin/io/airbyte/cdk/extensions/TestContext.kt

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.extensions
+
+class TestContext {
+    companion object {
+        val CURRENT_TEST_NAME: ThreadLocal<String?> = ThreadLocal()
```

Comment:
Convention in kotlin is to reserve upper-case snake case to `const val`s. I'm surprised your IDE didn't hint about this.

Reply:
from the [kotlin guide]([url](https://kotlinlang.org/docs/coding-conventions.html#property-names)): 
Names of constants (properties marked with const, or top-level or object val properties with no custom get function that hold deeply immutable data) should use uppercase underscore-separated (screaming snake case) names

I think this is what we're supposed to do

Reply:
TIL! Thanks